admin/worker/worker.go

```diff
@@ -94,6 +95,35 @@ func (w *Worker) schedule(ctx context.Context, name string, fn func(context.Cont
 	}
 }
 
+func (w *Worker) scheduleCron(ctx context.Context, name string, fn func(context.Context) error, cronExpr string) error {
+	loc, err := time.LoadLocation("America/Los_Angeles")
+	if err != nil {
+		return err
+	}
```

Comment:
I would recommend removing this (since we consistently use UTC on the server side), and instead passing the time zone directly in the crontab (which is supported in `robfig/cron/v3`). I think the format is: `CRON_TZ=America/Los_Angeles 0 0 * * 1` (but please verify)

Reply:
Sounds good! The format you provide here looks good to me, see reference:

https://pkg.go.dev/github.com/robfig/cron/v3#hdr-Usage

https://github.com/robfig/cron/blob/v3.0.1/parser.go#L95